addon/components/frost-toggle.js

```diff
@@ -130,8 +163,8 @@ export default Component.extend(PropTypeMixin, FrostEventsProxy, {
       }
     },
     /* eslint-enable complexity */
-    noop () {
-      // we have this method to avoid action undefined error
-    }
+
+    /** A no-op action to avoid an action undefined error */
```

Comment:
Single line comments should stay consistently `//`
Imho Function description comments  above the function [or really long comments] should have header+footer.